### PR TITLE
Update installation instructions in Getting Started

### PR DESCRIPTION
**Pull request by techpeace**
Just made some quick additions to the Getting Started docs to reflect what I needed to get the project up and running recently in my Rails 6 app.

### CODE REVIEW

**Comment by jonspalmer** [comment]
I think we need a different solution for making cors work. Its common to need in development BUT its important to frame those configs as not being for production etc.

File: docs/guide/getting-started.md [hunk 1]
```diff
@@ -14,12 +14,30 @@ nav_order: 1
...
    ```ruby
    require "view_component/storybook/engine"
    ```
-1. In`.gitignore`, add:
+1. In `config/environments/development.rb`, add:
+   ```ruby
+   # Allow use of Storybook
+   config.middleware.insert_before(0, Rack::Cors) do
+     allow do
+       origins "*"
```
Comment (Timmitry) [inline docs/guide/getting-started.md:32]:
```suggestion
       origins "*" do
```

Comment (techpeace) [reply docs/guide/getting-started.md:32]: Thanks for the catch!

**Comment by techpeace** [comment]
> I think we need a different solution for making cors work. Its common to need in development BUT its important to frame those configs as not being for production etc.

That makes sense, @jonspalmer. Would a comment or some additional context in the docs be sufficient?

**Comment by jonspalmer** [comment]
Yes I think some other section of the docs mentioning cors and dev setup would be more appropriate

File: docs/guide/getting-started.md (hunk 1 above)
Comment (cice) [reply docs/guide/getting-started.md]: Actually, without the `do` is correct, but there is an extra `end`

**Comment by jonspalmer** [comment]
Closing until we have a different approach. Happy re-open when appropraite

**Pull request closed without merging**